common-solidity/coordination-contract/contracts/CrosschainCoordinationV1.sol

```diff
@@ -30,6 +30,15 @@ contract CrosschainCoordinationV1 is CrosschainCoordinationInterface {
     // users can create a new sidechain. Only members of this sidechain can call addSidechain().
     uint256 public constant MANAGEMENT_PSEUDO_SIDECHAIN_ID = 0;
 
+    // Public key version number
+    uint private FIRST_PUBLIC_KEY = 0;
+    uint [] private publicKeyVersion;
+
+    // Public key & status
```

Comment:
Are these class / contract level fields used? They aren't inside the sidechain specific structure. As such, I expect they aren't used.

Reply:
Some fields have been deleted